CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+The documentation for each repository is available in the "**docs/**" directory of that repository.
+It is written in [Markdown format] and built using [MkDocs].
+
+## Testing Your Changes Locally
+
+We **strongly** encourage you to test your changes locally before contributing them.
+To do so, you first need to do three things; these are:
+
+1. Have MkDocs installed.
+2. Have the Laminas documentation theme available to the laminas-form directory.
+3. Update `mkdocs.yml` to build the docs using the documentation theme.
+
+To complete these steps, please follow the instructions below.
+
+1. [Install MkDocs].
+2. Install [the PyMdown Extensions].
+3. Clone the [documentation-theme repository] locally.
+    ```
+    git clone git@github.com:laminas/documentation-theme.git documentation-theme
+    ```
+4. Copy the theme directory from the cloned documentation-theme repository to the root directory of this project.
```

Comment:
I would suggest to use a Symlink instead of copy the theme directory and exclude it via Git.

Reply:
What about Windows, which doesn't support symlinks?

Reply:
I stand corrected. I just found out that versions Windows 7, 8, & 10 (I love counting in Microsoft) do support symlinks.